app/measure/measure.js

```diff
@@ -45,7 +45,10 @@ angular.module('Measure.Measure', ['ngRoute'])
       {
         userAcceptedDataPolicy: true,
         uploadworkerfile: "/libraries/ndt7-upload-worker.min.js",
-        downloadworkerfile: "/libraries/ndt7-download-worker.min.js"
+        downloadworkerfile: "/libraries/ndt7-download-worker.min.js",
+        metadata: {
+            client_name: "mlab-speedtest",
```

Comment:
To be more definitive, could we change this to "speed.measurementlab.net" or "speed-measurementlab-net"? I'm just wondering if "mlab-speedtest" might be too generic, in the case that we may launch some other speed test, like msak, at some other domain or location.